server/hotrod/src/main/java/org/infinispan/server/hotrod/util/BulkUtil.java

```diff
@@ -21,17 +17,18 @@
    public static final int LOCAL_SCOPE = 2;
 
    public static Iterator<byte[]> getAllKeys(Cache<byte[], ?> cache, int scope) {
```

Comment:
```scope``` is no longer used. A few years ago cache.size() and some other cache operations were only taking into account the local data, unless you were requesting 'global scope'. This was a very bad idea and was fixed long time ago.